hardhat.config.ts

```diff
@@ -298,6 +303,67 @@ task(
   }
 });
 
+task('build-timelock-authorizer-config', `Builds JSON file with Timelock Authorizer configuration`).setAction(
+  async (args: { verbose?: boolean }, hre: HardhatRuntimeEnvironment) => {
+    Logger.setDefaults(false, args.verbose || false);
+
+    if (hre.network.name === 'hardhat') {
+      logger.warn(`invalid network: ${hre.network.name}`);
+      return;
+    }
+
+    // Get active timelock authorizer task.
+    const tasks = Task.getAllTaskIds()
+      .filter((taskId) => taskId.includes('timelock-authorizer'))
+      .map((taskId) => new Task(taskId, TaskMode.READ_ONLY, hre.network.name))
+      .filter((task) => task.getStatus() === TaskStatus.ACTIVE);
+
+    if (tasks.length !== 1) {
+      const errorMsg = tasks.length === 0 ? 'not found' : 'is not unique';
+      logger.error(`Active timelock authorizer task ${errorMsg}`);
+      return;
+    }
+
+    saveTimelockAuthorizerConfig(tasks[0], hre.network.name);
+
+    logger.success(`Timelock Authorizer config JSON generated for network ${hre.network.name}`);
+  }
+);
+
+task(
+  'check-timelock-authorizer-config',
+  `Check whether the existing timelock authorizer configuration file is correct`
+).setAction(async (args: { verbose?: boolean }, hre: HardhatRuntimeEnvironment) => {
+  Logger.setDefaults(false, args.verbose || false);
+
+  if (hre.network.name === 'hardhat') {
+    logger.warn(`invalid network: ${hre.network.name}`);
+    return;
+  }
+
+  // Get active timelock authorizer task.
+  const tasks = Task.getAllTaskIds()
+    .filter((taskId) => taskId.includes('timelock-authorizer'))
+    .map((taskId) => new Task(taskId, TaskMode.READ_ONLY, hre.network.name))
+    .filter((task) => task.getStatus() === TaskStatus.ACTIVE);
+
+  if (tasks.length !== 1) {
+    const errorMsg = tasks.length === 0 ? 'not found' : 'is not unique';
+    logger.error(`Active timelock authorizer task ${errorMsg}`);
+    return;
+  }
+
+  const isConfigOk = checkTimelockAuthorizerConfig(tasks[0], hre.network.name);
+
+  if (!isConfigOk) {
```

Comment:
(I know this is still a draft...) But I think positive logic is simpler: if(isConfigOk) {...}